buildSrc/src/test/kotlin/com/example/model/Version.kt

```diff
@@ -30,12 +35,51 @@ class Version {
         fun fromJson(serializedObject: String): Version {
             try {
                 val jsonObject = JsonParser.parseString(serializedObject).asJsonObject
-                return Version()
+                val id = jsonObject.get("id").toString().let {
+                    Id.fromJson()
+                }
```

Comment:
This looks weird. Since the `Id` class is fully constant (and not nullable)  I'd expect this line to be 
```kotlin
val id = Id()
```

Reply:
yes...but wanted to be consistent with the other case where the value is optional and in the same time to actually check if there is actually a value for the `id` property in the serialized json. Otherwise I will introduce another case in this already too complicated logic.